enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/OCIItem.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.cloud.oracle;
+
+/**
+ * Represents Oracle Cloud Resource identified by Oracle Cloud Identifier (OCID)
+ * 
+ * @author Jan Horvath
+ */
+public class OCIItem {
```

Comment:
The `OCIItem` would then be the API of this module... but right now it would be _private API_ as the class isn't exposed.